build-utils/run_ci.sh

```diff
@@ -5,7 +5,6 @@ COMMAND=$1
 STUDY_KEY=$2
 BRANCH=$3
 TARGET_ENV=$4
-E2E_TEST_PARALLELISM=${5:-2}
```

Comment:
No change to build and deploy of different studies running parallel, right ? 
Ex: core-sdk change / merge triggering All studies deploy in parallel.

Reply:
no change to other like deploy jobs. only on playwright job.